lebedigital/calibration/calibrationWorkflow.py

```diff
@@ -117,6 +117,10 @@ def estimate_youngs_modulus(
     low_sigma = 0
     high_sigma = 0.005
 
+    # for reproducible results
+    seed =1
```

Comment:
This will use the same seed every time also for the real calibration.
This can't be a good solution.
What @joergfunger was asking, is to fix the seed only for the tests. This would require a possibility to pass the seed in the interface.

Reply:
Alright updated. please pull whenever the tests pass.